src/unk_0203F6C4.c

```diff
@@ -253,15 +253,15 @@ static BOOL ScrCmd_007(ScriptContext * param0);
 static BOOL ScrCmd_008(ScriptContext * param0);
 static BOOL ScrCmd_009(ScriptContext * param0);
 static BOOL ScrCmd_00A(ScriptContext * param0);
-static BOOL sub_0203F808(u16 param0, u16 param1);
+static BOOL Compare(u16 value0, u16 value1);
```

Comment:
Can we move this declaration out of the block of command handler declarations? Caught me off-guard seeing it here, at first.